src/test/scala/chiseltest/tests/ElementTest.scala

```diff
@@ -129,6 +130,40 @@ class ElementTest extends AnyFlatSpec with ChiselScalatestTester {
     }
   }
 
+  it should "peek on FixedPoint currently" in {
+    import chisel3.experimental.FixedPoint
+    test(new Module {
+      val io = IO(new Bundle {
```

Comment:
You might be able to use a MultiIOModule here and have all the IOs at the top level